torchvision/utils.py

```diff
@@ -180,6 +180,10 @@ def draw_bounding_boxes(
         raise ValueError("Pass individual images, not batches")
 
     ndarr = image.permute(1, 2, 0).numpy()
+    # allow single-channel-images
+    # shape: (1, H, W) with C = 1
```

Comment:
Isn't the shape (H, W, C) instead, since we permuted the dimensions in the line above?

Reply:
Yes, I realized the comment is not correct (only after the checks started running): it should be worded as below to avoid any confusion.

```python
# image tensor shape: (1, H, W) with C = 1
# ndarr.shape: (H, W, C)
```